packages/cache/src/types.js

```diff
@@ -0,0 +1,25 @@
+// @ts-check
+
+import '@agoric/store/exported.js';
+
+// Ensure this is a module.
+export {};
+
+/**
+ * @typedef {object} State
+ * @property {bigint} generation
+ * @property {any} value
```

Comment:
Question: any particular reason for using `any` over `unknown` here?

Reply:
If I use `unknown`, the type will propagate to the caller, which means they always have to inspect the value rather than just using it without inspection.